### PR TITLE
Bug 1431764 - Add shipitworker instance type

### PR DESCRIPTION
**Pull request by JohanLorenzo**

### CODE REVIEW

File: configs/securitygroups.yml [hunk 1]
```diff
@@ -556,6 +556,18 @@ pushapkworker:
     outbound:
         - include: global-any
 
+shipitworker:
+    description: security group for pushapkworker
```
Comment (nthomas-mozilla) [inline configs/securitygroups.yml]: Nit, pushapkworker

File: configs/shipitworker [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+    "us-east-1": {
+        "type": "shipitworker",
+        "domain": "srv.releng.use1.mozilla.com",
+        "ami": "ami-4dc07a26",
+        "subnet_ids": ["subnet-33a98358"], // TODO
+        "security_group_ids": ["sg-e3061897"],
```
Comment (nthomas-mozilla) [inline configs/shipitworker]: TODO should be on the security group rather than subnet, I think.

File: configs/securitygroups.yml (hunk 1 above)
Comment (JohanLorenzo) [reply configs/securitygroups.yml]: Oops. Done!

File: configs/shipitworker (hunk 2 above)
Comment (JohanLorenzo) [reply configs/shipitworker]: That's correct. I wasn't sure of the subnet, but NetOps was okay with it in [bug 1432838](https://bugzilla.mozilla.org/show_bug.cgi?id=1432838). I also confirm this security group is a new one, dedicated to shipit.

**Pull request merged**